app/components/form/WoW/OutOfStockForm/index.tsx

```
@@ -0,0 +1,108 @@
+}: {
+  defaultValues: Record<string, string | number | number[]>
+  onFormChange: (name: string, value: string) => void
+}) => {
+  // Helper function to safely convert value to array of numbers
+  const parseCategories = (value: string | number | number[]): number[] => {
+    if (Array.isArray(value)) return value
+    if (typeof value === 'string' && value.trim() !== '') {
+      return value
+        .split(',')
+        .map(Number)
+        .filter((n) => !isNaN(n))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)